### ISSUE TITLE
Sprint 20 (2018-05-25) Goals/Demo

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#744]
### Replication
* [ ] Implement zipmaker worker (#707)
* [x] Implement plexer worker (#709) blocked by (#713)
* [ ] Code md5 checksum storage (blocked by #673)

### Checksum Validation
* [x] perf stats into wiki
* [ ] error analysis -- are all errors the same type? which volumes, etc
* [x] finish running CV on all (15) volumes once

### Finish Robots:
* [x] Remove objects w/ druid: prefix (#722 )
* [x] Resolve issue with status change to "UNEXPECTED_VERSION" and confusing logging (#752)

### Stretch Goals
* [ ] (R) Result recorder worker (#711)

**Comment by jmartin-sul** [comment]
remaining unmet goals transferred to #806
